contracts/src/LightClient.sol

```diff
@@ -91,20 +94,22 @@ contract LightClient is Initializable, OwnableUpgradeable, UUPSUpgradeable {
     /// @param viewNum The latest view number of the finalized HotShot chain
     /// @param blockHeight The block height of the latest finalized block
     /// @param blockCommRoot The merkle root of historical block commitments (BN254::ScalarField)
-    /// @param feeLedgerComm The commitment to the fee ledger state (type: BN254::ScalarField)
-    /// @param stakeTableBlsKeyComm The commitment to the BlsVerKey column of the stake table
-    /// @param stakeTableSchnorrKeyComm The commitment to the SchnorrVerKey column of the table
-    /// @param stakeTableAmountComm The commitment to the stake amount column of the stake table
-    /// @param threshold The (stake-weighted) quorum threshold for a QC to be considered as valid
     struct LightClientState {
         uint64 viewNum;
         uint64 blockHeight;
         BN254.ScalarField blockCommRoot;
-        BN254.ScalarField feeLedgerComm;
+    }
+
+    /// @notice The finalized HotShot Stake state (as the digest of the entire HotShot state)
+    /// @param threshold The (stake-weighted) quorum threshold for a QC to be considered as valid
+    /// @param stakeTableBlsKeyComm The commitment to the BlsVerKey column of the stake table
+    /// @param stakeTableSchnorrKeyComm The commitment to the SchnorrVerKey column of the table
+    /// @param stakeTableAmountComm The commitment to the stake amount column of the stake table
+    struct StakeState {
+        uint256 threshold;
         BN254.ScalarField stakeTableBlsKeyComm;
         BN254.ScalarField stakeTableSchnorrKeyComm;
         BN254.ScalarField stakeTableAmountComm;
```

Comment:
naming nit:
```suggestion
    struct StakeTableState {
        uint256 threshold;
        BN254.ScalarField blsKeyComm;
        BN254.ScalarField schnorrKeyComm;
        BN254.ScalarField amountComm;
```

reason: 
- `StakeTableState` is clearer imo
- once we have the enclosing struct name to have the "stake table" prefix, we don't need it in the names of its fields, because `stakeTableState.blsKeyComm` already provides a clear context.

Reply:
btw, @alysiahuggins i'm addressing my own comments locally. just FYI to avoid redundant work

Reply:
done in [f7c5afa](https://github.com/EspressoSystems/espresso-sequencer/pull/1939/commits/f7c5afa0b52200c2e9ac8573ef6ca92c7aed7471)